trojsten/special/plugin_prask_5_1_1/core.py

```diff
@@ -111,11 +227,22 @@ def run(cls, x, try_count):
 
 class Level8(object):
 
-    TARGET = "8"
+    TARGET = "214365"
 
     @classmethod
     def run(cls, x, try_count):
-        return str(bin(x).count('1'))
+        digits = list(map(int, str(x)))
+        last = digits[0]
+        num = 1
+        res = ""
+        for i in (digits[1:] + [None]):
```

Comment:
Načo tu je ten `None`?

Reply:
Lebo python ma vo for cyklock ako porovnavanie "<" a ty chces ist od 1...n. Preto tam das nakoniec nejaky prvok